src/odo.ts

```diff
@@ -178,4 +179,12 @@ class OdoImpl implements Odo {
     public async execute(command: string, cwd?: string): Promise<CliExitData> {
         return this.cli.execute(command, cwd ? {cwd} : { });
     }
+
+    public async requireLogin(): Promise<boolean> {
+        const result: cliInstance.CliExitData = await this.cli.execute(
+            `odo version`, {}
+        );
+        // console.log(result.stdout.trim().split('\n')[1] === "Please log in to the cluster");
+        return result.stdout.trim().split('\n')[1] === "Please log in to the cluster";
```

Comment:
use `result.stdout.indexOf("Please log in to the cluster") > -1` instead.

Reply:
Fixed.